src/signalhandler.cc

```diff
@@ -145,7 +145,7 @@ void DumpTimeInfo() {
 void DumpStackFrameInfo(const char* prefix, void* pc) {
   // Get the symbol name.
   const char *symbol = "(unknown)";
-  char symbolized[1024];  // Big enough for a sane symbol.
+  char symbolized[1024] = {};  // Big enough for a sane symbol.
```

Comment:
I couldn't understand why this change is necessary. When Symbolize returns true, SymbolizeAndDemangle fills something to this buffer on both Windows and non-Windows?